src/js/reducers/notifications.js

```diff
@@ -0,0 +1,22 @@
+import AppConstants from '../constants/AppConstants';
+
+export default (state = {}, payload) => {
+    switch (payload.type) {
+        case(AppConstants.APP_NOTIFICATION_ADD): {
+            new Notification(
+                payload.notification.title,
+                {
+                    body: payload.notification.body,
+                    icon: payload.notification.icon,
+                    silent: true
+                }
+            );
```

Comment:
`New Notification` should be called in the action, not in the reducer, that's the purpose of action. Reducers are only used to change the state of the application